rekcurd_dashboard/apis/api_evaluation.py

```diff
@@ -245,9 +248,25 @@ def post(self, project_id: int, application_id: str):
         return response_body
 
 
-@evaluation_api_namespace.route('/projects/<int:project_id>/applications/<application_id>/evaluation_results/<int:eval_result_id>')
+@evaluation_api_namespace.route('/projects/<int:project_id>/applications/<application_id>/evaluation_results')
 class ApiEvaluationResults(Resource):
+    def get(self, project_id: int, application_id: str):
```

Comment:
Could you add a serializer?
```
@evaluation_api_namespace.expect(hoge)
@evaluation_api_namespace.marshal_with(fuga)
```

Reply:
fixed in https://github.com/rekcurd/dashboard/pull/105/commits/e0a21c7d65247b96070541c49a52202fbef180cc